vyper/ast/annotation.py

```diff
@@ -75,6 +78,12 @@ def visit_Module(self, node):
         return self._visit_docstring(node)
 
     def visit_FunctionDef(self, node):
+        if node.decorator_list:
+            # start the source highlight at `def` to improve annotation readability
+            decorator_token = node.decorator_list[-1].last_token
+            def_token = self._tokens.find_token(decorator_token, tokenize.NAME, tok_str="def")
+            node.first_token = def_token
```

Comment:
So, this will change the error message to point at the `def`?

Reply:
Yeah :)